contracts/smart-account/modules/SessionValidationModules/ERC20SessionValidationModule.sol

```diff
@@ -28,7 +28,13 @@ contract ERC20SessionValidationModule {
         bytes32 _userOpHash,
         bytes calldata _sessionKeyData,
         bytes calldata _sessionKeySignature
-    ) external pure returns (bool) {
+    ) external pure override returns (bool) {
+        require(
+            bytes4(_op.callData[0:4]) == EXECUTE_SELECTOR ||
+                bytes4(_op.callData[0:4]) == EXECUTE_OPTIMIZED_SELECTOR,
+            "ERC20SV Invalid Selector"
+        );
+
```

Comment:
Potential optimisation on line 54: != instead of >
https://twitter.com/filmakarov/status/1638540351815254016 😅

Reply:
done